Dockerfile

```diff
@@ -19,124 +19,34 @@
-
-RUN apt-get update && apt-get install --yes --no-install-recommends \
-    automake \
-    autopoint \
-    bc \
-    bison \
-    ca-certificates \
-    ccache \
-    flex \
-    gawk \
-    gettext \
-    git-core \
-    gperf \
-    intltool \
-    liblist-moreutils-perl \
-    liblzo2-dev \
-    libtool \
-    libxml-dom-perl \
-    libxml2-utils \
-    lua5.3 \
-    make \
-    mtd-utils \
-    net-tools \
-    nodejs \
-    openssh-client \
-    pkg-config \
-    python-pip \
-    python-setuptools \
-    python-wheel \
-    qemu-system-arm \
-    qemu-user \
-    rsync \
-    scons \
-    sudo \
-    texinfo \
-    u-boot-tools \
-    uuid-dev \
-    wget \
-    xutils-dev \
-    xz-utils \
-    zlib1g-dev \
-  && rm --recursive --force /var/lib/apt/lists/* \
-  && ln -sf /usr/bin/lua5.3 /usr/bin/lua \
-  && ln -sf /usr/bin/nodejs /usr/bin/node \
-  && ln -sf /bin/bash /bin/sh
-
-ENV HOME=/home/captain \
-  WSBU_C_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-gcc \
+ENV WSBU_C_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-gcc \
   WSBU_CXX_COMPILER=/opt/linaro/bin/arm-linux-gnueabihf-g++ \
   WSBU_EMULATOR=/usr/bin/qemu-arm \
   QEMU_LD_PREFIX=/opt/linaro/arm-linux-gnueabihf/libc \
   CMAKE_TOOLCHAIN_FILE=/opt/toolchain-linaro-armhf.cmake
 
-COPY toolchain.cmake "${CMAKE_TOOLCHAIN_FILE}"
-RUN sed -i 's;@GCC_INSTALL_ROOT@;/opt/linaro;' "${CMAKE_TOOLCHAIN_FILE}"
-
-RUN git clone https://github.com/wsbu/linaro-release.git \
-    --branch gcc-linaro-4.9.4-2017.01-x86_64_arm-linux-gnueabihf \
-    --depth 1 \
-    /opt/linaro \
-  && rm -rf /opt/linaro/.git
-
-RUN git clone https://github.com/wsbu/mtd-utils.git \
-    --branch v2.0.1 \
-    --depth 1 \
-    /src \
-  && cd /src \
-  && ./autogen.sh \
-  && ./configure \
-  && make \
-  && make install \
-  && cd - \
-  && rm -rf /src
-
-RUN git clone https://github.com/wsbu/cross-browser.git \
-    --branch x419_z1 \
-    --depth 1 \
-    /src \
-  && cd /src/x/xc \
-  && gcc -c xc.c -O2 \
-  && gcc -o xc xc.o -O2 \
-  && cp --force xc /bin \
-  && strip /bin/xc \
-  && mkdir --parents /lib/crossbrowser \
-  && cp --archive /src/x/lib/*.js /lib/crossbrowser \
-  && cp --archive /src/x/lib/old/*.js /lib/crossbrowser \
-  && cd - \
-  && rm -rf /src
-
-RUN wget --quiet -O /tmp/cmake.sh https://cmake.org/files/v3.10/cmake-3.10.1-Linux-x86_64.sh \
-  && sh /tmp/cmake.sh --prefix=/usr/local --exclude-subdir --skip-license \
-  && rm /tmp/cmake.sh
-
-# Install Conan
-RUN pip install conan==1.0.4
 ENV CONAN_CMAKE_TOOLCHAIN_FILE="${CMAKE_TOOLCHAIN_FILE}" \
-  CONAN_PRINT_RUN_COMMANDS=1 \
   CC="${WSBU_C_COMPILER}" \
   CXX="${WSBU_CXX_COMPILER}"
+
+# Replace native-oritented build configurations
+COPY toolchain.cmake "${CMAKE_TOOLCHAIN_FILE}"
 COPY conan/sitara_profile "${HOME}/.conan/profiles/sitara"
 COPY conan/settings.yml "${HOME}/.conan/settings.yml"
 COPY conan/registry.txt "${HOME}/.conan/registry.txt"
-RUN ln -s "${HOME}/.conan/profiles/sitara" "${HOME}/.conan/profiles/default"
```

Comment:
Please leave this symlink in place. Without it, it makes it way too easy to accidentally build x86 binaries with the toolchain-linaro docker image. It might make sense to run `mv "${HOME}/.conan/profiles/default" "${HOME}/.conan/profiles/native"` first though.

Reply:
Ah, I somehow missed the "sitara", and just thought it was recreating the link created by native. Using the force...